src/components/Home.jsx

```diff
@@ -1,3 +1,32 @@
+import SingleArticle from "./ArticleCard";
+import * as API from '../utils/api';
+import { useState, useEffect } from 'react';
+
 export default function Home () {
-    
+
+    // States To Track User's Filtering - If any of the sort/order/topic change then re-render the articles displayed
+    const [displayedArticles, setDisplayedArticles] = useState([]); 
+    const [sort, setSort] = useState('created_at');
+    const [order, setOrder] = useState('DESC');
```

Comment:
you aren't using any of these states yet be careful of adding things before they are needed